CallsheetLocalizations/pl.xcloc/Localized Contents/pl.xliff

```diff
@@ -861,18 +904,22 @@
       </trans-unit>
       <trans-unit id="No TV show pins found." xml:space="preserve">
         <source>No TV show pins found.</source>
+        <target>Brak przypiętych seriali.</target>
         <note/>
       </trans-unit>
       <trans-unit id="No credits found." xml:space="preserve">
         <source>No credits found.</source>
+        <target>Brak udziałów filmowych.</target>
```

Comment:
"Brak filmów w których występuje"? Jest bezosobowa, a "Udziały" to po polsku bardziej akcje (shares/stock).